### DISCUSSION TITLE
Trying to setup autonomous flying with Inav and MAVProxy

### DISCUSSION THREAD

**Discussion by elleuc4** [iNavFlight/inav#8700] [Q&A]
Hi, I'm trying to understand how to control my fixed wing plane through a Raspberry Pi running MAVProxy and connected to UART.
I've already setup in the configurator everything required to send and receive commands, when running MAVProxy I can read the telemetry from the flight controller

My goal is to have the plane follow fixed waypoints by itself, without using a joystick.

Is there documentation for this? right now I'm stuck at arming the system, when I input "arm safetyoff / arm throttle" nothing seems to happen.

I also tried to connect with Mission Planner and I can use the joystick to move the ailerons, and that's it.

**Comment by sensei-hacker** [comment]
Sorry this didn't get answered in a timely manner!

"My goal is to have the plane follow fixed waypoints by itself, without using a joystick."

If that was indeed the goal, see the documentation on missions / waypoint mode:
https://github.com/iNavFlight/inav/wiki/INAV-Missions

No Raspberry Pi or Mavproxy is needed to achieve that goal.
